microcosm_flask/tests/conventions/test_query.py

```diff
@@ -120,7 +120,7 @@ def test_swagger(self):
                             "type": "string",
                         },
                         {
-                            "required": False,
+                            "required": True,
```

Comment:
Add another case here to make sure that:
* non-required fields are marked as such
* required fields are similarly marked